src/packages/excalidraw/README_NEXT.md

```diff
@@ -599,6 +602,18 @@ Indicates whether to bind keyboard events to `document`. Disabled by default, me
 
 Enable this if you want Excalidraw to handle keyboard even if the component isn't focused (e.g. a user is interacting with the navbar, sidebar, or similar).
 
+### onLibraryChange
+
+Ths callback if supplied will get triggered when the library is updated and has the below signature.
+
+<pre>
+(items: <a href="https://github.com/excalidraw/excalidraw/blob/master/src/types.ts#L200">LibraryItems</a>) => void | Promise<any>
+</pre>
+
+### id
+
+The unique Id for the excalidraw component. This can be used to identify the excalidraw component for example importing the library items to the correct excalidraw component when you have multiple excalidraw components rendered on the same page.
```

Comment:
```suggestion
The unique id of the excalidraw component. This can be used to identify the excalidraw component, for example if you want to import library items only to the component which triggered it (when you have multiple excalidraw components rendered on the same page).
```

Reply:
The `only` here can be confusing since if there are multiple excal and the host doesn't check in which to import then it will be importing it to the instance(depending on whats the ref defaulting to) and not all containers.

Reply:
Yes, is is confusing — though that applies to the original too. I guess we should explain how if we haven't done so anywhere (don't recall).